cmd/report.go

```diff
@@ -18,246 +18,24 @@
 package cmd
 
 import (
-	"fmt"
```

Comment:
Cleaning up `cmd/report.go` as this code was not maintained and doesn't work with a basic use case. We are trying to redefine report functionality. Will add implementation as needed.